packages/desktop-client/src/components/reports/graphs/BarGraph.tsx

```diff
@@ -130,13 +130,10 @@ const CustomLegend = ({ active, payload, label }: CustomLegendProps) => {
 type BarGraphProps = {
   style?: CSSProperties;
   data;
-  groupBy;
+  groupBy: string;
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review (if it exists :^)  -->
**👏 praise:** ‏thanks for adding missing types